plugins/cnfreg/config.go

```diff
@@ -55,5 +51,9 @@ func (p *Plugin) loadConfig() (*Config, error) {
 	}
 
 	p.Log.Debugf("CnfRegistry config found: %+v", cfg)
+	var durZeroVal time.Duration
+	if cfg.CnfDiscoveryTimeout != durZeroVal {
+		p.Log.Warn("Option CnfDiscoveryTimeout is deprecated and setting it has no effect")
```

Comment:
Maybe also telling what is the new way of discovery and why the timeout was deprecated. Some words that the discovery is running continuously(by filesystem-driven events) from now and the timeout is meaningless now because of that. This could help for someone that is migrating versions to understand what is going on.

Reply:
Done.